app/controllers/support_interface/performance_controller.rb

```diff
@@ -10,10 +10,10 @@ def course_options
 
     def courses_dashboard; end
 
-    def performance_dashboard
-      raise ArgumentError unless params[:year].in?([nil, '2020', '2021'])
+    def service_performance_dashboard
+      year = params[:year] if ['2020', '2021'].include?(params[:year])
```

Comment:
```suggestion
      year = params[:year] if %w[2020 2021].include?(params[:year])
```
I think rubocop may complain about that

Reply:
It did! Fixed 😀